### ISSUE TITLE
[YSQL] ALTER OPERATOR SET <define>

### ISSUE THREAD

**Issue by ddorian** [yugabyte/yugabyte-db#2711]
Add support for alter operator

**Comment by irizzant** [comment]
+1 For this I can't restore my dump because of this issue

**Comment by ddorian** [comment]
@irizzant can you paste the full query ?

**Comment by irizzant** [comment]
@ddorian in reality I think this is issued through the activation of `cube` estension:
```
pg_restore: creating EXTENSION "cube"
pg_restore: [archiver (db)] Error while PROCESSING TOC:
pg_restore: [archiver (db)] Error from TOC entry 3; 3079 436183759 EXTENSION cube 
pg_restore: [archiver (db)] could not execute query: ERROR:  ALTER OPERATOR SET <define> not supported yet
HINT:  Please report the issue on https://github.com/YugaByte/yugabyte-db/issues
    Command was: CREATE EXTENSION IF NOT EXISTS cube WITH SCHEMA public;
```

**Comment by frozenspider** [comment]
@irizzant: Turns out, `ALTER OPERATOR SET <define>` wasn't the only thing necessary for `cube` extension to work, tracking it here: #3628
I'm working on it, should be ready soon.

**Comment by irizzant** [comment]
@frozenspider Thanks for the update I subscribed #3628 too

**Comment by frozenspider** [comment]
Hey @irizzant, this specific issue has been resolved in a6d2160. While this does in principle allow you to use a `cube` extension (i.e. `CREATE EXTENSION cube`), note that it doesn't work exactly as it does in vanilla Postgres yet - specifically, you won't be able to apply `ORDER BY` to a `cube`-type column yet.
